### DISCUSSION TITLE
Japanese AtlasOS Wallpaper V1.2

### DISCUSSION THREAD

**Discussion by Justikodere** [Atlas-OS/Atlas#130] [Community Artwork]
![My AtlasOS Wallpaper V1 2](https://user-images.githubusercontent.com/78940734/159589823-aa702a1d-add4-42fb-aae3-81ac83eb6859.png)

**Comment by namke1303** [comment]
good walpaper brother <3

**Comment by diegotopow** [comment]
Awesome. Could you make a dark theme wallpaper for this one?

**Reply by hubman-star** [reply]
invert that wallpaper and done, there's your dark theme wallpaper.
